Add less documentation

kev-zheng opened the pull request:
# Advanced HW 5 submission

Closes #403

Name: Kevin Zheng
Uniqname: kevzheng

cameron-gagnon commented on _commands/basics/less.md:
```diff
@@ -3,22 +3,69 @@
+Typing `/` within `less` allows you to forward search using regex
+and `?` allows you to backward search
+pressing `n` goes to the next item
+~~~ bash
```
You need an extra newline before the "~~~" so that it will render as a code block.

cameron-gagnon commented on _commands/basics/less.md:
```diff
@@ -3,22 +3,69 @@
+
 
-##### Break it down
+#Options:
```
You need an extra space between the "#" and "options" so that it renders as a bold header.

cameron-gagnon commented on _commands/basics/less.md:
```diff
@@ -3,22 +3,69 @@
+| **-X**  |      Contents of file remain on screen after exiting.      |
 
-##### Break it down
+#Commands:
```
Need an extra space between the two here as well.

cameron-gagnon commented on _commands/basics/less.md:
```diff
@@ -3,22 +3,69 @@
+#Options:
 
-#### Example command
+| Options |                         Description                        |
```
You need "~~~" surrounding the tables here to render them as a table.

Also, the "**" be removed from the commands, they don't render as bold, and it's a bit distracting from the actual content.

cameron-gagnon commented on _commands/basics/less.md:
```diff
@@ -3,22 +3,69 @@
 
-##### Break it down
+#Commands:
+| Command         |                              Description                             |
```
Need a newline before the "~~~" that will need to be inserted here as well.

cameron-gagnon commented on _commands/basics/less.md:
```diff
@@ -3,22 +3,69 @@
+| **:n**          |                               Next file                              |
+| **:p**          |                             Previous file                            |
+| **h**           |             Help. This is presented with less, q to quit.            |
+| **q**           |                                 Quit                                 |
```
Need "~~~" at the end as well.

cameron-gagnon commented on _commands/basics/less.md:
```diff
@@ -3,22 +3,69 @@
+#Commands:
+| Command         |                              Description                             |
+|-----------------|:--------------------------------------------------------------------:|
+| **Space bar**   |                               Next Page                              |
```
Could you include the reference (either a link or another command to run) to get this information in the tables? That way people can dig deeper if they want on their own.

cameron-gagnon commented on _commands/basics/less.md:
```diff
@@ -3,22 +3,69 @@
 ~~~
 
-<!--more-->
+<!--less-->
```
This needs to stay as "more" because the templating checks for this comment to decide where to split the short reference page example and the fuller examples.

cameron-gagnon commented:
Also, can you delete the `_commands/tools/less.md` file? It's interfering with the `_commands/basics/less.md` file which is where you made your changes.

kev-zheng commented:
I've just added the formatting changes and removed tools/less.md. On my end, the tables won't render if I include the '~~~' on my markdown previewer; I added them anyway.
